### PR TITLE
[WIP] Trying to improve the code-climate rating

### PR DESCRIPTION
**Pull request by abellotti**
- playing first with the VM resource actions

### CODE REVIEW

File: app/controllers/api/vms_controller.rb [hunk 1]
```diff
@@ -263,6 +173,19 @@ def set_miq_server_resource(type, id, data)
 
     private
 
+    def exec_vm_resource_action(verb, action, method, type, id = nil)
```
Comment (imtayadeway) [inline app/controllers/api/vms_controller.rb]: Does there need to be a default argument for id?

Also, I'd prefer no abbreviations in these method names.

5 arguments is a lot of arguments. Did you consider using kwargs (or a hash) to better convey what they do from the calling end?

Comment (Fryguy) [inline app/controllers/api/vms_controller.rb]: I agree with @imtayadeway but let's handle in a followup

**Comment by Fryguy** [comment]
@abellotti Bump 🐢  (just remove the WIP...JUST DO IT!!!)

**Comment by abellotti** [comment]
will revisit later.

**Pull request closed without merging**